optimum/exporters/neuron/config.py

```diff
@@ -53,3 +56,39 @@ class TextAndVisionNeuronConfig(NeuronConfig):
     """
 
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, DummyVisionInputGenerator, DummyBboxInputGenerator)
+
+
+class DecoderNeuronConfig(ExportConfig):
```

Comment:
It should inherint from `NeuronConfig` imo.

Reply:
I considered this, but NeuronConfig is too specific to the models that were previously integrated: none of its methods nor class constants are actually required by the export code for causal LM models.
For the causl LM models, all we need is:
- some way to identify the corresponding transformers_neuronx model class,
- some way to identify the kwargs that need to be passed when calling from_pretrained on that class.

Reply:
Then I would move this to the `neuron/base.py` file since it can be considered as a base class in that sense.

Reply:
@dacorvo @michaelbenayoun if we finally decide to move this class to base.py, call it rather `NeuronDecoderConfig` .

Reply:
Done